### PR TITLE
Add FAQ about coordinator address

### PR DESCRIPTION
**Pull request by RiccardoMasutti**

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -940,6 +940,18 @@ When all peers are registered, then the signing phase is done within a couple of
 Summing up: the faster peers register in the CoinJoins, the faster the mixes are.
 :::
 
+:::details
+### What is coordinator's address?
+
+The coordinator participates in every CoinJoin.
+For transaparency reasons, the coordinator address is reused for every CoinJoin.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
For transparency reasons, the coordinator address is used for every CoinJoin.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -940,6 +940,18 @@ When all peers are registered, then the signing phase is done within a couple of
...
+
+The last address used by Wasabi coordinator is `bc1qa24tsgchvuxsaccp8vrnkfd85hrcpafg20kmjw`.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
The current address used by the Wasabi coordinator is `bc1qa24tsgchvuxsaccp8vrnkfd85hrcpafg20kmjw`.
```

File: docs/using-wasabi/AddressReuse.md [hunk 3]
```diff
@@ -48,6 +48,8 @@ The idea of coordinator address reuse is transparency.
 We could give up transparency here to confuse less sophisticated observers by creating a new coordinator address for every CoinJoin. 
 It’s a different tradeoff, not an improvement.
 
+You can check every address used by Wasabi coordinator [here](/FAQ/FAQ-UseWasabi.md#what-is-coordinators-address).
```
Comment (yahiheb) [inline docs/using-wasabi/AddressReuse.md]:
```suggestion
You can check every address used by the Wasabi coordinator [here](/FAQ/FAQ-UseWasabi.md#what-is-coordinators-address).
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 4]
```diff
@@ -940,6 +940,18 @@ When all peers are registered, then the signing phase is done within a couple of
 Summing up: the faster peers register in the CoinJoins, the faster the mixes are.
 :::
 
+:::details
+### What is coordinator's address?
+
+The coordinator participates in every CoinJoin.
+For transparency reasons, the coordinator address is used for every CoinJoin.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
And for transparency reasons, the same coordinator address is used.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 5]
```diff
@@ -940,6 +940,18 @@ When all peers are registered, then the signing phase is done within a couple of
 Summing up: the faster peers register in the CoinJoins, the faster the mixes are.
 :::
 
+:::details
+### What is coordinator's address?
+
+The coordinator participates in every CoinJoin.
```
Comment (MaxHillebrand) [inline docs/FAQ/FAQ-UseWasabi.md]: I wouldn't say the coordinator "participates" in every round, as he has neither inputs - nor equal value outputs. He gets paid in every round, yes. 

So I suggest changing to `The coordinator gets paid in every CoinJoin.`

File: docs/FAQ/FAQ-UseWasabi.md [hunk 6]
```diff
@@ -940,6 +940,18 @@ When all peers are registered, then the signing phase is done within a couple of
 Summing up: the faster peers register in the CoinJoins, the faster the mixes are.
 :::
 
+:::details
+### What is coordinator's address?
```
Comment (MaxHillebrand) [inline docs/FAQ/FAQ-UseWasabi.md]: I would change to
`What is the coordinator address?`

Maybe 
`What is the coordinator's address?`

**Comment by RiccardoMasutti** [comment]
@MaxHillebrand ready

**Pull request merged**